builder/docker/artifact_import.go

```diff
@@ -30,19 +30,12 @@ func (a *ImportArtifact) Id() string {
 }
 
 func (a *ImportArtifact) String() string {
-	tags := a.StateData["docker_tags"]
-	if tags == nil {
-		return fmt.Sprintf("Imported Docker image: %s", a.Id())
+	tags, _ := a.StateData["docker_tags"].([]string)
```

Comment:
here we could do:
```suggestion
	var tags []string
	switch t := a.StateData["docker_tags"].(type) {
	case []string:
	  tags = t
	case []interface{}:
	  tags = toStringSlice(t)
	case nil:
	  // nothing
	default:
	  panic("something")
	}
```

and same in the following example; just to be safe 🤔

Reply:
done! ✔️